Python/Product/Debugger.Concord/Guids.cs

```diff
@@ -0,0 +1,56 @@
+// Python Tools for Visual Studio
```

Comment:
Is this copy of Guids.cs needed? It looks like you already link the version at ../debugger/guids.cs

Reply:
Yep, just something I overlooked.